libs/features/links.js

```diff
@@ -0,0 +1,28 @@
+let fetched = false;
+let noFollowData = null;
+
+const getNoFollowLinks = async (path) => {
```

Comment:
I'd probably change the name of the function now.

Reply:
done!